addons/controllers/antrea/antreaconfig_controller.go

```diff
@@ -167,10 +194,163 @@ func (r *AntreaConfigReconciler) ReconcileAntreaConfigNormal(
 		return err
```

Comment:
The implementation of ReconcileAntreaConfigDataValue should also be modified to pass Antrea-NSX configs to values.yaml like below:

```
antrea:
  # … antrea related configs
antrea-nsx:
  enable: true    # set the values according to AntreaConfig.spec.antreaNsx.enable
  bootstrapFrom:  # bootstrapFrom either providerRef or inline configs
    providerRef:    # set the values when Antrea-NSX is bootstrap from NSXServiceAccount. Fill the NSXServiceAccount name created by addon controller to below resource name.
      apiVersion: nsx.vmware.com/v1alpha1
      kind: NSXServiceAccount
      name: test-cluster1-antrea
    inline:  # set the values according to AntreaConfig.spec.antreaNsx.bootstrapFrom.inline is set
      nsxManagers: [sampleNSXIP1, sampleNSXIP2, sampleNSXIP3]
      clusterName: test-cluster1
      nsxCert:
        tls.crt: … # base64 encoded certificate
        tls.key: … # base64 encoded certificate
  config:
    infraType: vSphere # or VMC, AWS, Azure
    # … other configs such as log verbosity, security policy reconciliation timeout
```

Reply:
Antrea Package needs to be updated to support the new fields in values.yaml, too. I'm now sure if existing Antrea Package can ignore these new fields. We can have a try on manually editing the values.yaml in workload cluster and see if Antrea PackageInstall reconciles successfully.

Reply:
OK

Reply:
done